nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -2872,7 +2872,7 @@ provides several functions for evaluating the same conditions against groups of
 |=======================================================================================
 | Expression | Value
 | `${allMatchingAttributes("[ax].*"):substringBefore(" "):join("-")}` | `hello-good`
-| `${allAttributes("abc", "xyz"):join(" now")}` | `hello world nowgood bye world now`
+| `${allAttributes("abc", "xyz"):join(" now ")}` | `hello world now good bye world`
```

Comment:
There is  a space between "now" and "good bye" in the correct output because there is a space at the end of the `join` parameter. It reads more easily now as "hello world now good bye world" rather than the more awkward "hello world nowgood bye world" with no spaced between "now" and "good".